src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+class WinningLotto(
+    private val winningNumbers: Set<LottoNumber>,
+    private val bonusNumber: LottoNumber
+) : Lotto(winningNumbers) {
```

Comment:
Lotto를 상속받아 WinningLotto를 표현해주셨네요👍
WinningLotto가 Lotto의 하위 타입인 만큼 winningNumbers라는 새로운 멤버 프로퍼티는 없어도 될 것 같아요.
```suggestion
class WinningLotto(
    winningNumbers: Set<LottoNumber>,
    private val bonusNumber: LottoNumber
) : Lotto(winningNumbers)
```